test/network/2020-03-01/armnetwork/zz_generated_models.go

```diff
@@ -13470,8 +13469,8 @@ type VirtualNetworkPeeringPropertiesFormat struct {
 	RemoteAddressSpace *AddressSpace `json:"remoteAddressSpace,omitempty"`
 
 	// The reference to the remote virtual network. The remote virtual network can be in the same or different region (preview). See here to register for the
-	// preview and learn more (
-	// https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-create-peering [https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-create-peering]).
+	// preview and learn more
+	// (https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-create-peering).
```

Comment:
Similar question about surrounding `()`.

Reply:
That's coming from the swagger